content/about/agreements.md

```diff
@@ -91,15 +103,31 @@ We agree to annual reviews, to document goals and assess trajectory, by filling
 
 ### Trainnee Agreements
 
-#### Annual Productivity
 
 Depending on the "stage" of our careers, and our career objectives, we have different milestones and deliverables. But annual productivity is a crucial component of any career development.  We therefore conditionally agree to the following:
 
+#### Publications
+
 - *MSE students*: submit a first author publication before graduation 
 - *PhD students*: submit a first author publication once per each of the last three years of the PhD
-- *Postdocs*: submit a first author publication once per year **and** be key personnel on at least one grant application (make sure you have a target solicitation >2 months in advance)
+- *Postdocs*: submit a first author publication once per year
 - *Research staff*: take ownership of a new development year-long development "project"
 
+#### Conferences
+
+Each team member will present at >=1 conference per year to get feedback, network, and professional development.  Any conferences one desires to attend is fine, conditional on
+- you will present your work via some mechanism (posters, talks, or boothing) are basically required, and
+- for conferences with proceedings, a "submission ready" draft is provided >= 1 month prior to submission deadline, or
+- for conferences with posters, a "presentation ready" poster is provided >= 2 weeks prior to the event.
+
+
+
+
+#### Grants
+
+Each postdoc will apply for funding each year, targeting a solicitation >2 months in advance
+ 
+ 
 #### Research Prioritees
 
 In general, the research priority for any student is to be working towards writing a paper within the year.  Other priorities include preparing for conferences where you are presenting.  And then your PI may have other more immediate responsibilities to attend to, and may ask you to produce a figure/result/etc. with some urgency.  In general, prioritize what is most important, rather than most urgent.  However, deadlines with dates are infrequent, and typically strict.  In other words, if we have an impending deadline, agreeing to it means that all research associated with that product is the top priority.  Thus, unless otherwise stated, if a PI asks you to produce *x*, interpret that as the most urgent and top priority. 
```

Comment:
I find this part confusing. "if a PI asks you to produce *x*" happens almost daily, and I don't think it is always the result of an urgent deadline.

Reply:
This section could be stated as "prioritize what is most important, rather than most urgent, unless your PI needs something now"?

Reply:
make a PR